<launch>    
  <!-- Including the base launch file with parameters -->
  <include file="$(find mercator_sensor_fusion_ros_pkg)/launch/base_launch_files/arena_poses_publisher.launch">
    <arg name="frame_id" value="odom"/>
    <arg name="child_frame_ids" value="['base_link_23', 'base_link_31']"/>
    <arg name="pose_topic" value="/arena_poses"/>
  </include>

    <!-- Including the base launch file with parameters -->
  <include file="$(find mercator_sensor_fusion_ros_pkg)/launch/base_launch_files/robot_to_arena_poses_transformer.launch">
    <arg name="node_name" value="robot_to_arena_poses_transformer"/>
    <arg name="new_poses_frame_id" value="odom"/>
    <arg name="frame_id" value="odom"/>
    <arg name="child_frame_id" value="base_link_22"/>
    <arg name="robot_odom_frame_id" value="odom"/>
    <arg name="robot_frame_id" value="robot"/>
    <arg name="cam_poses" value="/fused_poses"/>
    <arg name="lidar_poses" value="/donothing"/>
    <arg name="cam_poses_odom" value="/fused_poses_arena"/>
    <arg name="lidar_poses_odom" value="/donothingtoo"/>
    <arg name="tf" value="tf"/>
  </include>

    <!-- Including the base launch file with parameters -->
  <include file="$(find mercator_sensor_fusion_ros_pkg)/launch/base_launch_files/cost_between_poses.launch">
    <arg name="experiment_topic" value="/fused_poses_arena"/>
    <arg name="ground_truth_topic" value="/arena_poses"/>
  </include>
</launch>